fleets/day2/chart-templates/akri-dashboard-extension/fleet.yaml

```diff
@@ -2,7 +2,7 @@ defaultNamespace: cattle-ui-plugin-system
 
 helm:
   releaseName: akri-dashboard-extension
-  chart: "oci://registry.suse.com/edge/3.2/akri-dashboard-extension-chart"
-  version: "302.0.0+up1.2.1"
+  chart: "oci://registry.suse.com/edge/charts/akri-dashboard-extension"
+  version: "303.0.0+up1.2.1"
```

Comment:
The [version in Factory](https://src.opensuse.org/suse-edge/Factory/src/branch/main/release-manifest-image/release_manifest.yaml#L170) is `303.0.2+up1.3.1`